Sasquatch/SasquatchObjC/AppCenterDelegateObjC.m

```diff
@@ -361,4 +361,15 @@ - (void)replaceDocumentWithPartition:(NSString *_Nonnull)partitionName
               completionHandler:completionHandler];
 }
 
+- (void)readDocumentWithPartition:(NSString *_Nonnull)partitionName
+                       documentId:(NSString *_Nonnull)documentId
+                     documentType:(Class)documentType
+                completionHandler:(void (^)(MSDocumentWrapper *))completionHandler {
+  [MSData readDocumentWithID:documentId documentType:documentType partition:partitionName completionHandler:completionHandler];
+}
+
+- (NSString *)kMSDataUserDocumentsPartition {
```

Comment:
Maybe it would be better to make it a constant

Reply:
Moved to `Constants.h`